templates/project/psalm.xml.twig

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+-->
+<psalm xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="https://getpsalm.org/schema/config" errorLevel="2" findUnusedPsalmSuppress="true" resolveFromConfigFile="true" xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"{% if file_exists(project_dir ~ '/psalm-baseline.xml') %} errorBaseline="psalm-baseline.xml"{% endif %}>
```

Comment:
The easier solution would be to use level 3 on this conf, but it would say that some projects will regress from 2 to 3. Do we consider this as a "big" lost ? @sonata-project/contributors

Reply:
BlockBundle is now level 3

Reply:
I would make it configurable, default value 2, and overriden for the needed one with higher level.